concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -132,6 +144,7 @@ disable_pipeline_locking: ${DISABLE_PIPELINE_LOCKING:-}
 datadog_api_key: "${datadog_api_key:-}"
 datadog_app_key: "${datadog_app_key:-}"
 aiven_api_token: ${aiven_api_token:-}
+pagerduty_spike_integration_key: "${pagerduty_spike_integration_key:-this-is-not-a-pagerduty-key}"
```

Comment:
Alex pointed out we shouldn't have the word 'spike' in there anymore, as we are doing this for real.